lab_2/main.py

```diff
@@ -34,7 +48,17 @@ def fill_lcs_matrix(first_sentence_tokens: tuple, second_sentence_tokens: tuple)
     :param second_sentence_tokens: a tuple of tokens
     :return: a lcs matrix
     """
-    pass
+    if not isinstance(first_sentence_tokens, tuple) or not isinstance(second_sentence_tokens, tuple) or \
+            None in first_sentence_tokens or None in second_sentence_tokens:
+        return []
+    lcs_matrix = create_zero_matrix(len(first_sentence_tokens), len(second_sentence_tokens))
+    for row, word_1 in enumerate(first_sentence_tokens):
+        for column, word_2 in enumerate(second_sentence_tokens):
+            if word_1 == word_2:
```

Comment:
what happens if row == 0 or column == 0?

Reply:
if row == 0 I get the first row of the matrix
if column == 0 I get the first element in the current row

Reply:
I mean, what happens here `lcs_matrix[row - 1][column - 1] + 1` and here `max((lcs_matrix[row][column - 1], lcs_matrix[row - 1][column]))`

Reply:
Oh, okay. If row  ==  0 or column == 0, we get the last row or column in the matrix because indexes become -1
However, it is not a problem because we work with the first elements when the last elements are just 0, we haven't done anything to them yet. 
For example, if word_1 == word_2 and lsc_matrix[-1][-1]  we will still get 1